.github/workflows/rust.yml

```
@@ -10,7 +10,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  MINIMUM_RUST_VERSION: 1.74.0
+  MINIMUM_RUST_VERSION: 1.81.0
```

The minimum rust version is still 1.74 in the version of the sdk in use in this change, so I don't think we need to change this. The minimum rust version named here, checks that the examples all still build with the minimum rust version of the SDK.
```suggestion
  MINIMUM_RUST_VERSION: 1.74.0
```